tutorials/histfactory/ModifyInterpolation.C

```diff
@@ -88,16 +84,11 @@ void ModifyInterpolationForAll(RooWorkspace* ws, int code){
 
 void ModifyInterpolationForSet(RooArgSet* modifySet, int code){
 
-  TIter it = modifySet->createIterator();
-  RooRealVar* alpha = nullptr;
-  while((alpha=(RooRealVar*)it.Next())){
-    TIter serverIt = alpha->clientIterator();
-    TObject* tempObj = nullptr;
-    while((tempObj=serverIt.Next())){
-      FlexibleInterpVar* flex = dynamic_cast<FlexibleInterpVar*>(tempObj);
+  for (RooAbsArg *alpha : *modifySet) {
+    for (auto *flex : dynamic_range_cast<FlexibleInterpVar*>(alpha->clients())) {
       if(flex){
          flex->printAllInterpCodes();
-         flex->setInterpCode(*alpha,code);
+         flex->setInterpCode(static_cast<RooAbsReal &>(*alpha),code);
```

Comment:
Thanks!! It all looks good to me. Was just not sure why alpha was before RooRealVar and now RooAbsReal, but I don't know the details or differences, so just probably ignore this comment. :)

Reply:
Thanks for the review! I don't know why it was `RooRealVar` before, which was unnecessarily specific. I just used `RooAbsReal` now, because this is what the interface asked for:
https://root.cern.ch/doc/master/classRooStats_1_1HistFactory_1_1FlexibleInterpVar.html#a2070e04fa485299295f7299ee58d2b3e